blackcellmagic.py

```diff
@@ -37,7 +37,13 @@ def black(self, line, cell):
         args = magic_arguments.parse_argstring(self.black, line)
         line_length = args.line_length
         if cell:
-            formated = format_str(src_contents=cell, line_length=line_length)
+            try:
+                formated = format_str(src_contents=cell, line_length=line_length)
+            except TypeError:
```

Comment:
Should you not pass the current correct argument (`mode`) first - in the `try` statement - and only then fall back to the deprecated `line_length `?

Reply:
Correct